src/protocol/crsf_uart.c

```diff
@@ -62,6 +66,9 @@ u8 crsf_crc8(const u8 *ptr, u8 len) {
 static u8 telemetryRxBuffer[TELEMETRY_RX_PACKET_SIZE];
 static u8 telemetryRxBufferCount;
 
+static s32 offset = 0;
+static u32 updateInterval = CRSF_FRAME_PERIOD;
```

Comment:
Since updateInterval is used in serial_cb it needs to move outside #if HAS_EXTENDED_TELEMETRY, otherwise modular builds fail.

Reply:
moved